types/react-native/index.d.ts

```diff
@@ -8747,7 +8747,7 @@ export namespace Animated {
     export const timing: (value: AnimatedValue | AnimatedValueXY, config: TimingAnimationConfig) => CompositeAnimation;
 
     interface TimingAnimationConfig extends AnimationConfig {
-        toValue: number | AnimatedValue | { x: number; y: number } | AnimatedValueXY;
+        toValue: number | AnimatedValue | { x: number; y: number } | AnimatedValueXY | AnimatedInterpolation;
```

Comment:
I'm not sure that this properly reflects the type in the RN source? https://github.com/facebook/react-native/blob/0ee5f68929610106ee6864baa04ea90be0fc5160/Libraries/Animated/src/animations/TimingAnimation.js#L21

Reply:
I'm not sure where the discrepancy in the typing is coming from. Do you have any ideas?

Reply:
@nabati Shall we open an issue to react-native first? To ask them to fix their flow types?

Reply:
@chinesedfan Seeing as it’s been a while since OP responded (and I’m unsure how the PR didn’t get closed by the bot), perhaps you could ask upstream and report back?

Reply:
I made a PR now, https://github.com/facebook/react-native/pull/27558.

Reply:
The above mentioned PR has been merged, btw.

Reply:
Thanks for the bump, @Sakarisson 👍